app/assets/stylesheets/v3/table.css.scss

```diff
@@ -35,8 +35,10 @@
   line-height: $item_height;
   position: relative;
 }
-.table__row.is-clickable:hover {
-  box-shadow: 0 0 1px 1px rgba(100, 100, 100, 0.1);
+.table__rows, .table__row {
```

Comment:
Selector `table__rows` should be written in lowercase with hyphens<br>Selector `table__row` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line